entry_types/scrolled/package/src/frontend/playerControls/themes/ControlBarThemeTransparent.module.css

```diff
@@ -0,0 +1,15 @@
+.foreground {
```

Comment:
What was the issue with using child selectors in a single CSS module?

Reply:
I kept getting into the situation where I needed styles from one css module for use in another... themes are now removed, though.